docs/controls/WebView-known-issues.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Known Issues for WebView control for Windows Forms and WPF
+author: normesta
+description: This guide highlights known limitations with the current release of the WebView control for Windows Forms and WPF applications.
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, WebView, Windows Forms, WPF, known issues, release notes
+---
+
+# Known Issues of the WebView control for Windows Forms and WPF applications
+
+This document describes issues and limitations in this release of the **WebView** control.
+
+## WebView control members
+
+The **WebView** control does not implement these methods of the [IWebViewControl](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol).
+
+* [NavigateToLocalStreamUri](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.navigatetolocalstreamuri)
+
+* [NavigateWithHttpRequestMessage](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.navigatewithhttprequestmessage)
+
+* [CapturePreviewToStreamAsync](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.capturepreviewtostreamasync)
+
+* [CaptureSelectedContentToDataPackageAsync](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.captureselectedcontenttodatapackageasync)
+
+* [BuildLocalStreamUri](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.buildlocalstreamuri)
+
+The **WebView** control implements these events of the [IWebViewControl](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol), but they don't work as expected.
+
+* [LongRunningScriptDetected](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.longrunningscriptdetected)
+
+* [UnsafeContentWarningDisplaying](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unsafecontentwarningdisplaying)
+
+* [UnsupportedUriSchemeIdentified](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unsupportedurischemeidentified)
+
+* [UnviewableContentIdentified](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unviewablecontentidentified)
+
+* [NewWindowRequested](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.newwindowrequested)
+
+## Rendering and layout
+
+* The **WebView** control doesn't appear at the correct scale when users move the host application between monitors that have different screen resolutions.
+
+* The **WebView** control doesn't appear as expected on some high-resolution displays.
+
+* The [Control.Bounds](https://msdn.microsoft.com/library/system.windows.forms.control.bounds.aspx) property is not supported.
+
+## Performance
+
+* **WebView** controls in WPF applications have longer load times than Windows Forms applications.
+
+* If the web view hangs or stops working, other applications that use the same underlying process (Win32WebViewHost) will hang or stop working as well.
```

Comment:
You can see how bad this is by using one of the samples at `\\scratch2\scratch\rimuri\WebView\Samples` and trying to watch something on Netflix. It's an easy 30-second pause where you have no idea if it is just taking a long time to do something or is broken.